all/generate_gateway.sh

```diff
@@ -0,0 +1,300 @@
+#!/bin/bash
+
+set -e
+
+HTTP_PORT=8080
+
+printUsage() {
+  echo "Generates a docker image of a grpc-gateway server for a given service proto."  
+  echo
+  echo "The container will listen on port ${HTTP_PORT} for HTTP traffic, and proxy requests"
+  echo "to the gRPC service. In addition, it will serve a Swagger definition of its API at"
+  echo "/swagger.json"
+  echo
+  echo "Usage: generate_gateway.sh -f <service.proto> -p service_package -s ServiceName"
```

Comment:
-p is service_package here, put below it's port.

Reply:
we need to support multiple proto files. this bit me. it is important that all proto files are passed to `protoc` at once. `entrypoint.sh` was recently changed to better handle this.

Reply:
Fixed yo.